src/pages/index.tsx

```diff
@@ -42,7 +43,10 @@ export default function Home() {
         <MobileNavigationSpacer>
           <Toolbar variant="dense" />
         </MobileNavigationSpacer>
-        <BottomAppBar />
+
+        <Box display={['none', 'block']}>
+          <BottomAppBar />
+        </Box>
```

Comment:
This gets rid of:

- Changing settings
- Slow mode
- Adding Torrents
- The inspector icon

You can still access the inspector by double-tapping a torrent